modules/tinymce/src/plugins/accordion/main/ts/core/Keyboard.ts

```diff
@@ -25,11 +25,14 @@ const setup = (editor: Editor): void => {
   });
 
   editor.on('keydown', (e) => {
-    if (e.keyCode === VK.BACKSPACE) {
+    if (e.keyCode === VK.BACKSPACE || e.keyCode === VK.DELETE) {
```

Comment:
This breaks Ctrl-Delete or Ctrl-Backspace so you can't delete words inside a details or summary element.

Reply:
The only time you need to run this custom code is when you are in the intersection between summary and the details body so I guess you could try to detect that and only execute the delete when that happens since when you hit ctrl-backspace in say `<summary>foo</summary><p>|bar</p>` then the delete acts like a regular delete since the block boundary is being treated as a word boundary. You could make some form of edge function but all that could be handled with the CaretFinder stuff but that is in the core.

Reply:
I've moved the BACKSPACE and DELETE handlers to the core (`selection/DetailsElement.ts`). Ended up using `CaretFinder` and a couple of `e.preventDefault()` methods. Let me know what you think, please!

Reply:
Delete doesn't seem to work in the branch and the backspace still merges things into the summary. I think for checking if the caret is in the first or last position you could do:
```
// Is caret at beginning of summary
CaretFinder.firstPositionIn(summary).exists((pos) => pos.isEqual(CaretPosition.fromRangeStart(editor.selection.getRng())));

// Is caret at end of summary
CaretFinder.lastPostionIn(summary).exists((pos) => pos.isEqual(CaretPosition.fromRangeStart(editor.selection.getRng())));

// Is caret just before summary in details element
CaretFinder.lastPostionIn(summary).exists((pos) => pos.isEqual(CaretPostion.prevPosition(editor.selection.getRng()).exists((next) => next.isEqual(pos))));
```
The CaretPosition `isAtStart` will just check if the position is the first in the current element or text node not the first in the current block scope.

Reply:
Weirdly, but everything worked in FF. I can reproduce the problem in Chrome though. Fixed the code with more peculiar `CaretFinder` checks.

Reply:
Ok seems to work now in Chrome nice. :+1: